experimental/parser/testdata/parser/method/incomplete.proto.stderr.txt

```diff
@@ -28,17 +28,17 @@ error: expected exactly one type in method parameter list, got 0
 24 |     rpc Bar5() returns (stream foo.Bar);
    |             ^^
 
-error: missing method return type in service method
-  --> testdata/parser/method/incomplete.proto:25:5
+error: expected exactly one type in method parameter list, got 0
+  --> testdata/parser/method/incomplete.proto:25:13
    |
 25 |     rpc Bar6() returns;
-   |     ^^^^^^^^^^^^^^^^^^^
+   |             ^^
 
-error: expected exactly one type in method parameter list, got 0
+error: missing method return type in service method
   --> testdata/parser/method/incomplete.proto:25:13
    |
 25 |     rpc Bar6() returns;
-   |             ^^
+   |             ^^ expected type in `(...)` after this
```

Comment:
This seems rather wrong. It's point to before the returns keyword. If we don't have easy access to the bare "return" token, we should add a TODO (so we can improve this) but also try to massage the text so it's not as jarring or confusing.

Maybe we could even indicate the semicolon as the location for this diagnostic and say that we expected the return type "before this"?

Reply:
This is straight-up a parser bug. I've fixed it.